lib/modules/platform/local/index.ts

```diff
@@ -0,0 +1,124 @@
+/* istanbul ignore file */
```

Comment:
This file is just boilerplate to satisfy types, and two functions return static objects